content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -276,6 +345,16 @@ Before using the integration, specify the following information on the contact:
    .. image:: brazil/contact-fiscal-configuration.png
       :alt: Contact fiscal configuration.
 
+#. Configure the following extra :guilabel:`Fiscal Information` if you are going to issue NFS-e:
+
+   - Add the :guilabel:`Fiscal Position` for :ref:`Avatax Brazil <brazil/fiscal-positions>`
```

Comment:
Same as above regarding the brackets + add GUI labels where needed (see my other comment on the same topic above)